src/components/headers/HeaderWrapper.tsx

```diff
@@ -13,12 +13,19 @@ export interface IHeaderWrapperProps extends ITabsHeaderProps, React.ClassAttrib
 export class HeaderWrapper extends React.Component<IHeaderWrapperProps, {}> {
 
   private getClasses(): string {
-    return classNames('flex',
-      'flex-center space-between',
+    return classNames(
+      'flex',
+      'flex-center',
+      'space-between',
       'mod-header-padding',
       'header-height',
-      'pb2',
-      this.props.classes);
+      {
+        'pt1': !!this.props.tabs,
+        'pb2': !!this.props.tabs,
+        'p1': !this.props.tabs,
```

Comment:
This will add padding left and right too which will be overriden by `mod-header-padding`. I'd prefer `pl1` and `pr1`

Reply:
nice catch, i actually need pt1 and pb1

Reply:
fixed